bootstrap.sh

```diff
@@ -1 +1,36 @@
 #!/bin/bash
+
+# Create namespaces
+echo "Creating namespaces..."
+kubectl create namespace mysql || echo "Namespace 'mysql' already exists."
+kubectl create namespace todoapp || echo "Namespace 'todoapp' already exists."
+
+# Deploy MySQL resources
+echo "Deploying MySQL resources..."
+kubectl apply -f ./.infrastructure/namespace.yml
```

Comment:
Applying `namespace.yml` is redundant since namespaces are already created at the beginning of the script. Consider removing this line to avoid potential conflicts or unnecessary operations.